File: Aula 2/Exemplo 1.py
class Usuario:
   # Método Construtor: Inicializa os atributos (Estado)
    def __init__(self, nome, email):
      self.nome = nome
      self.email = email
    
    # Método de Classe (Comportamento)

    def exibir_perfil(self):
       print(f"Usuário: {self.nome}; Contato: {self.email}.")

# Instanciando dois objetos distintos na memórias
usuario1 = Usuario("Ana Silva", "ana@email.com")
usuario2 = Usuario("Carlos Moura", "carlos@email.com")

# Invocando comportamentos

usuario1.exibir_perfil()
usuario2.exibir_perfil()

File: Aula 2/Exemplo 1.js
class UsuarioJS {
    // Palavra-chave 'constructor' para inicialização
    constructor(nome, email) {
        this.nome = nome;
        this.email = email;
    }
    exibirPerfil() {
        console.log(`Usuário: ${this.nome}; Contato: ${this.email}.`);
    }
}

// Instanciação utilizando a palavra-chave 'new'
const user1 = new UsuarioJS('João Mendes', 'joao@email.com');
user1.exibirPerfil();